.mk/k8s.mk

```diff
@@ -16,7 +16,7 @@ K8S_CONF_DIR = k8s/conf
 
 # Deployments - common
 DEPLOY_TRACING = jaeger
-DEPLOY_WEBHOOK = admission-webhook
+DEPLOY_WEBHOOK = admission-webhook nsm-coredns nsm-monitor
```

Comment:
I don't quite understand why we are adding these here.  Could you explain?

Reply:
OK... realized I need to read more further down to understand... will try that :)